desktop/mac/release-notes/index.md

```diff
@@ -72,6 +72,14 @@ blog post [Apache Log4j 2 CVE-2021-44228](https://www.docker.com/blog/apache-log
   - Ensure your version of `systemd` supports cgroupv2. [It must be at least `systemd` 247](https://github.com/systemd/systemd/issues/19760#issuecomment-851565075). Consider upgrading any `centos:7` images to `centos:8`.
   - Containers running `systemd` need the following options: [`--privileged --cgroupns=host -v /sys/fs/cgroup:/sys/fs/cgroup:rw`](https://serverfault.com/questions/1053187/systemd-fails-to-run-in-a-docker-container-when-using-cgroupv2-cgroupns-priva).
 
+### Known issue
+
+Docker Dashboard incorrectly displays the container memory usage as zero.
+You can use the CLI command [docker
+stats](../engine/reference/commandline/stats/) as a workaround to view the
+actual memory usage. See
```

Comment:
I think the link is broken, also perhaps "You can use the `docker stats` command...

```suggestion
You can use the [`docker stats`](../../../engine/reference/commandline/stats.md)
command on the command line as a workaround to view the actual memory usage. See
```